### PR TITLE
misc: Update start guide in README

### PR DESCRIPTION
**Pull request by vincent-pochet**
This PR is related to https://github.com/getlago/lago/issues/369

It updates the `Run the app` section of the README to details the required steps for the first start on a fresh install 

It closes https://github.com/getlago/lago/pull/370

### CODE REVIEW

**Review by brunomiguelpinto — APPROVED** [review]

**Comment by Citrullin** [comment]
@brunomiguelpinto @vincent-pochet No, this doesn't fix #370

Just checked
```
Schema migrations table does not exist yet.
```

Please follow the steps from #369 for a fresh installation. It required to remove the mounted docker volumes. 
I even just checked with a whole fresh VM. Just to be sure this really is equivalent to a fresh installation.

lago_test
lago_test2

File: README.md [hunk 1]
```diff
@@ -99,8 +99,15 @@ cd lago
 echo "LAGO_RSA_PRIVATE_KEY=\"`openssl genrsa 2048 | base64`\"" >> .env
 source .env
 
-# Start
-docker-compose up
+# Start the api
+docker-compose up -d api api-worker
```
Comment (Citrullin) [inline README.md]: `docker-compose` is v1. Please change to `docker compose`

File: README.md [hunk 2]
```diff
@@ -99,8 +99,15 @@ cd lago
...
+
+# Wait for all migrations to be performed
+# Check status with:
+docker compose exec api rails db:migrate:status
+
+# Start the front and the clock
+docker-compose up -d front api-clock
```
Comment (Citrullin) [inline README.md]: Same here. Should be `docker compose`

**Review by Citrullin — CHANGES_REQUESTED** [review]

**Comment by Citrullin** [comment]
Just for you in order to reproduce. I rented a fresh Ubuntu 22.04 VM on Scaleway. Installed docker and docker compose.

```bash
root@lago-test:~/lago# docker --version
Docker version 27.0.3, build 7d4bcd8
root@lago-test:~/lago# docker compose version
Docker Compose version v2.28.1
```

**Comment by vincent-pochet** [comment]
> Just for you in order to reproduce. I rented a fresh Ubuntu 22.04 VM on Scaleway. Installed docker and docker compose.
> 
> ```shell
> root@lago-test:~/lago# docker --version
> Docker version 27.0.3, build 7d4bcd8
> root@lago-test:~/lago# docker compose version
> Docker Compose version v2.28.1
> ```

It's a sequence issue. I'm adding `docker compose exec api rails db:create:primary` after the initial start of api / api-worker to make sure that the database is created

**Pull request merged**